phpstan.neon.dist

```diff
@@ -0,0 +1,10 @@
+includes:
+    - phpstan-baseline.neon
+
+parameters:
+    level: 5
```

Comment:
Well, I suggest running at the highest possible level, as the higher the level, the better phpstan is at catching mistakes. The level configuration is a balance between catching mistakes (the higher the better for that) and not being too annoying (when introducing phpstan in a new project, higher levels can be hard to reach)

Reply:
@stof I totally agree :+1: 

As written in the description, I checked multiple repos below github.com/symfony/ and all that I checked used level 5, so I thought it was some kind of convention for symfony repos :D 

Updated to max level in 761acfdd07b56feb2eb8de0803acc8c8c8c3f456 (which also results in a quite a bit larger baseline file)